Gemfile.lock

```diff
@@ -59,7 +59,7 @@ GIT
       activemerchant (~> 1.34)
       acts_as_list (= 0.2.0)
       awesome_nested_set (= 2.1.5)
-      aws-sdk (~> 1.11.1)
+      aws-sdk (~> 1.3.4)
```

Comment:
that is quite a big change. Have you checked why it happened? I find it weird to have these Gemfile.lock changes at this point. The revision moves us a bunch of commits forward only, right? :thinking:

Reply:
this is just the upload/download of images from s3 right?
we are moving one commit forward only.
I dont understand why this happens.
shall I lock 1.11 on our side, in our Gemfile?

Reply:
> this is just the upload/download of images from s3 right?

I guess so as it's defined as a `spree_core` dependency. I can't think of any other Spree feature that might need aws.

> I dont understand why this happens.

Me neither. I'll try to reproduce it in my machine.

Reply:
Ok, same outcome on my machine. For some reason when resolving spree's dependencies again (through a `bundle update spree` bundler thinks these versions are better suited 🤷‍♀️ . So locking aws-sdk to 1.11 or above sounds good IMO.

Reply:
I tried to do it myself but I found out why bundler was doing it in the first place:

```
Bundler could not find compatible versions for gem "aws-sdk":
  In Gemfile:
    aws-sdk (>= 1.11.1)

    spree was resolved to 2.0.4, which depends on
      spree_core (= 2.0.4) was resolved to 2.0.4, which depends on
        aws-sdk (~> 1.3.4)
```

So it turns out the Gemfile.lock was wrong. Maybe a wrongly solved merge conflict?

Nothing to do here then.

Reply:
ah, nice, thanks for having a look!
spree 204 depends on 1.3.4 in spree repo [here](https://github.com/spree/spree/blob/v2.0.4/core/spree_core.gemspec#L23) and in ofn spree repo [here](https://github.com/openfoodfoundation/spree/blob/2-0-4-stable/core/spree_core.gemspec#L23).
What I just found is that our current spree branch step-6a is on 1.11.1 since a [long time ago](https://github.com/openfoodfoundation/spree/commit/6c98889f7453983da874b89d842e1945438bb6d0).
So I think we need to open a new PR in ofn/spree/2-0-4 and change it to depend on 1.11.1
Then update this PR to use that new revision of ofn/spree/2-0-4 and keep 1.11.1. Makes sense?

Reply:
ok, I went for it.
There's only the update of the spree revision in this PR now.
I'll update aws-sdk in a separate PR.

Reply:
Glad that you carried on with the investigation. Sounds good :+1:

Reply:
yeah, this is the only one breaking a fresh bundle install